docker-for-windows/index.md

```diff
@@ -48,148 +45,78 @@ and make sure `docker` commands are working properly.
-    init version: 949e6fa
-    Security Options:
-     seccomp
-      Profile: default
-    Kernel Version: 4.9.12-moby
-    Operating System: Alpine Linux v3.5
-    OSType: linux
-    Architecture: x86_64
-    CPUs: 2
-    Total Memory: 1.934 GiB
-    Name: moby
-    ID: BM4O:645U:LUS6:OGMD:O6WH:JINS:K7VF:OVDZ:7NE4:ZVJT:PSMQ:5UA6
-    Docker Root Dir: /var/lib/docker
-    Debug Mode (client): false
-    Debug Mode (server): true
-     File Descriptors: 13
-     Goroutines: 21
-     System Time: 2017-03-02T16:59:13.417299Z
-     EventsListeners: 0
-    Registry: https://index.docker.io/v1/
-    Experimental: true
-    Insecure Registries:
-     127.0.0.0/8
-    Live Restore Enabled: false
-    ```
-
-    >**Note**: The outputs above are examples. Your output for commands like
-    > `docker version` and `docker info` varies depending on your product
-    > versions.
-
-3.  Run `docker run hello-world` to test pulling an image from Docker Hub and starting a container.
+2.  Run `docker run hello-world` to test pulling an image from Docker Hub and starting a container.
 
     ```none
-    PS C:\Users\jdoe> docker run hello-world
-
-    Hello from Docker.
+    PS C:\Users\Gordon> docker run hello-world
+
+    docker : Unable to find image 'hello-world:latest' locally
+    ...
+
+    latest:
+    Pulling from library/hello-world
+    ca4f61b1923c:
+    Pulling fs layer
+    ca4f61b1923c:
+    Download complete
+    ca4f61b1923c:
+    Pull complete
+    Digest: sha256:97ce6fa4b6cdc0790cda65fe7290b74cfebd9fa0c9b8c38e979330d547d22ce1
+    Status: Downloaded newer image for hello-world:latest
+
+    Hello from Docker!
     This message shows that your installation appears to be working correctly.
-
-    To generate this message, Docker took the following steps:
-    1. The Docker client contacted the Docker daemon.
-    2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
-    3. The Docker daemon created a new container from that image which runs the executable that produces the output you are currently reading.
-    4. The Docker daemon streamed that output to the Docker client, which sent it to your terminal.
+    ...
     ```
-
-4.  Try something more ambitious, and run an Ubuntu container with this command.
+3.  List the hello-world image:
 
     ```none
-    PS C:\Users\jdoe> docker run -it ubuntu bash
+    docker image ls
     ```
 
-    This downloads the `ubuntu` container image and start it. Here is the output of running this command in a powershell.
+4.  Run an image of the Ubuntu OS:
 
     ```none
-    PS C:\Users\jdoe> docker run -it ubuntu bash
-
-    Unable to find image 'ubuntu:latest' locally
-    latest: Pulling from library/ubuntu
-    5a132a7e7af1: Pull complete
-    fd2731e4c50c: Pull complete
-    28a2f68d1120: Pull complete
-    a3ed95caeb02: Pull complete
-    Digest: sha256:4e85ebe01d056b43955250bbac22bdb8734271122e3c78d21e55ee235fc6802d
+    PS C:\Users\Gordon> docker run -it ubuntu bash
```

Comment:
I would add `--rm` `docker run --rm -it ...`.  Unless of course you depend on it for the removal of the containers below.

Reply:
wilco